specs/commandline/list.md

```diff
@@ -29,6 +29,7 @@ Aliases:
 Flags:
   -d, --debug             debug mode
   -h, --help              help for list
+      --oci-layout        [Preview] whether the artifact is OCI image layout
```

Comment:
```suggestion
      --oci-layout        [Preview] use OCI image layout for storage
```

Reply:
This flag is to specify the signing artifact type, but this sentence `use OCI image layout for storage` sounds like storing the artifact or signature in OCI image layout.